src/analytics/segment.js

```diff
@@ -33,15 +34,16 @@ const extraProperties = store => {
   const { localeIdentifier, preferredLanguages } = Locale.constants();
   const language = languageSelector(state);
   const devices = knownDevicesSelector(state);
-  const lastDevice = devices[devices.length - 1];
+  
+  const lastDevice = lastSeenDeviceSelector(state) || devices[0];
```

Comment:
It should be `devices[devices.length - 1]` not `devices[0]` no? Otherwise we are getting the first one as a fallback

Reply:
I didn't commit it. 100%